### ISSUE TITLE
"TypeError: timestamps.includes is not a function" when opening list of entities with timestamps disabled

### ISSUE THREAD

**Issue by AntonShan** [strapi/strapi#7163]
**Describe the bug**
I have a colelction type with timestamps disabled. When opening the list of entities I'm having an error `TypeError: timestamps.includes is not a function`.

**Steps to reproduce the behavior**
1. Create a type with no timestamps
2. Open admin panel and open list of entites of newly create type

**Expected behavior**
List of entities should open without errors

**Screenshots**
![image](https://user-images.githubusercontent.com/4890060/88423467-e4111980-cdfc-11ea-9fbb-0cc6f7e22a98.png)

**Code snippets**
```
{
  "kind": "collectionType",
  "collectionName": "menu",
  "info": {
    "name": "menu"
  },
  "options": {
    "increments": true,
    "timestamps": false
  },
  "attributes": {
    ...
  }
}

```
If applicable, add code samples to help explain your problem.

**System**

- Node.js version: v12.16.3
- NPM version: 6.14.4
- Strapi version: 3.1.1
- Database: SQLite
- Operating system: Windows 10 Pro

**Additional context**
It looks like the problem is with FilterPicker component of Content Manager plugin. It tries to take value of `options.timestamps` on line 37 but its value is `false` which leads to error on line 43.
The correct line should probably look like this:
`  const timestamps = get(schema, ['options', 'timestamps'], []);`
but I'm not sure about it
There's no such problem if type has timestamps enabled.

**Comment by lauriejim** [comment]
Hello! I just tried and everything is working well.
Did you have this error with a fresh app or after a migration?

**Comment by AntonShan** [comment]
> Did you have this error with a fresh app or after a migration?
I had this error after migration.

I think it was something with the build process because I was working on my application and I had to switch between branches with different strapi versions several time and after one of switches bug just disappeared. And it looks we can close the issue.

This is especially weird because I ran `yarn build --clean` after migration.
